test/functional/content-view/create_test.rb

```diff
@@ -0,0 +1,125 @@
+require File.join(File.dirname(__FILE__), '../test_helper')
+
+describe 'content-view create' do
+  before do
+    @cmd = %w(content-view create)
+    @base_params = ["--organization-id=#{org_id}", "--name=#{name}"]
+  end
+  let(:error_heading) { "Could not create the content view" }
+  let(:name) { 'test-cv' }
+  let(:org_id) { 1 }
+  let(:sanitize) do
+    # Workaround for deprecation of --id in favor of --uuid in Katello 2.5
+    ->(result) { result.err.gsub!(/^Warning: .*?\n/, '') }
+  end
+  let(:product) do
+    { 'name' => 'product-1', 'id' => 1 }
+  end
+  let(:repositories) do
+    [
+      {'name' => 'repo-1', 'id' => '1'},
+      {'name' => 'repo-2', 'id' => '2'},
+      {'name' => 'repo-3', 'id' => '3'}
+    ]
+  end
+
+  it 'creates the content view with repositories specified by ids' do
+    wanted = repositories.take(2)
+    ids = wanted.map { |repo| repo['id'] }
+    params = %W(--repository-ids=#{ids.join(',')})
+
+    api_expects(:content_views, :create, 'Create content view') do |par|
+      par['name'] == name &&
+        par['repository_ids'] == ids &&
+        par['organization_id'] == org_id
+    end
+
+    expected_result = success_result("Content view created\n")
+    result = run_cmd(@cmd + @base_params + params)
+    sanitize.call(result)
```

Comment:
Just for the record:
After a further investigation it turned out the printed warning is a bug. The warning should be displayed only when users really pass `--id`. Currently it occurs every time the content-view command is loaded.